routers/web/repo/actions/actions.go

```diff
@@ -245,6 +245,11 @@ func List(ctx *context.Context) {
 		return
 	}
 
+	if err := loadIsRefDeleted(ctx, runs); err != nil {
+		ctx.ServerError("LoadIsRefDeleted", err)
```

Comment:
Maybe there's no need to return server error here.

Reply:
3a0676099d